WcaOnRails/app/models/user.rb

```diff
@@ -293,6 +283,44 @@ def after_confirmation
     end
   end
 
+  def software_team?
+    team_member?('software')
+  end
+
+  def results_team?
+    team_member?('results')
+  end
+
+  def wrc_team?
+    team_member?('wrc')
+  end
+
+  def wdc_team?
+    team_member?('wdc')
+  end
+
+  def team_member?(team)
+    self.team_members.where(team_id: ( self.teams.find_by_friendly_id(team) ) ).any?(&:current_member?)
+  end
+
+  def was_team_member?(team)
```

Comment:
I guess we could, but...why exactly? Looks much more elegant to have the method instead of checking that
```
some_member.end_date != nil && some_member.end_date < Date.today
```